sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -285,7 +285,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
       Azure::Nullable<DateTime> NextAllowedQuotaDowngradeTime;
       Azure::Nullable<DateTime> DeletedOn;
       int32_t RemainingRetentionDays = int32_t();
-      Azure::Nullable<AccessTier> AccessTier; // The access tier of the share.
+      Azure::Nullable<AccessTier> Tier; // The access tier of the share.
```

Comment:
`Tier` is ambiguous as a field name. I recommend going the other way and renaming the other service `AccessTier` to be consistent so the name is self-documenting/descriptive. I believe that's what .NET does.

Reply:
@Jinming-Hu What do you think. This is aligning with Blob.

Reply:
Okay, 

```suggestion
      Azure::Nullable<Models::AccessTier> AccessTier; // The access tier of the share.
```